src/_guides/api-basics/docs-as-code-elevating-api-documentation-with-developer-workflows.md

```diff
@@ -0,0 +1,115 @@
+
+## What is Docs as Code?
+
+Docs as Code applies software development best practices—such as version control, automated testing, and continuous integration—to documentation. This approach enables technical writers and developers to collaborate more effectively, ensuring that API documentation remains up-to-date, consistent, and of high quality. For large organizations with multiple teams and shared technical writers, Docs as Code offers a structured framework to blend developer-generated documentation and improvements and additional documentation produced by technical writers. 
+
+## Continuous Delivery Principles for Docs
+
+Applying continuous delivery principles to documentation brings numerous advantages, such as improved collaboration, automation, and version control. Below are the key components that drive the Docs as Code methodology.
+
+### Version Control
+
+Version control systems, such as [Git](https://git-scm.com/), provide a structured way to track changes in documentation. By managing documentation in a repository alongside the codebase, teams can:
+
+- Maintain a historical record of changes
+- Revert to previous versions when necessary
+- Enable concurrent editing and review processes
+- Facilitate branching and merging to manage documentation updates
+
+This version-controlled approach ensures that documentation evolves alongside the API, reducing the risk of outdated or inconsistent information.
+
+_Are you new to git? Check out our article that provides an [introduction to git concepts](https://docs.bump.sh/guides/technical-writing/primitive-concepts-git/)._
+
+### Automated Doc Testing
+
+Automated testing is a key aspect of ensuring documentation quality. While commonly associated with software testing, it can also be applied to technical documentation through:
+- **Linting tools** (e.g., [Vale](https://vale.sh/), [Markdownlint](https://github.com/DavidAnson/markdownlint)) to enforce style guides and formatting consistency
+- **Link checkers** to validate internal and external links
+- **Schema validation tools** to ensure API documentation aligns with actual API definitions
+- **Spell checkers and grammar checkers** to catch errors before publication
+- **OpenAPI linters** (e.g., [Vacuum](https://bump.sh/blog/api-linting-with-vacuum)) to enforce compliance with your organization’s API style guide. 
```

Comment:
Bit of a funny one but the author of the tool says it should be `vacuum` not `Vacuum`, and we've gone with that elsewhere.
```suggestion
- **OpenAPI linters** (e.g., [vacuum](https://bump.sh/blog/api-linting-with-vacuum)) to enforce compliance with your organization’s API style guide. 
```